econml/dml.py

```diff
@@ -271,29 +276,91 @@ def predict(self, X):
         super().__init__(model_y=FirstStageWrapper(model_y, is_Y=True),
                          model_t=FirstStageWrapper(model_t, is_Y=False),
                          model_final=FinalWrapper(),
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
                          random_state=random_state)
 
+    @property
+    def original_featurizer(self):
+        return super().model_final._original_featurizer
+
     @property
     def featurizer(self):
+        # NOTE This is used by the inference methods and has to be the overall featurizer. intended
+        # for internal use by the library
         return super().model_final._featurizer
 
     @property
     def model_final(self):
+        # NOTE This is used by the inference methods and is more for internal use to the library
+        return super().model_final._model
+
+    def model_cate(self):
+        """
+        Get the fitted final CATE model.
+
+        Returns
+        -------
+        model_cate: object of type(model_final)
+            An instance of the model_final object that was fitted after calling fit which corresponds
+            to the constant marginal CATE model.
+        """
         return super().model_final._model
 
     @property
     def models_y(self):
+        """
+        Get the fitted models for E[Y | X, W].
+
+        Returns
+        -------
+        models_y: list of objects of type(`model_y`)
+            A list of instances of the `model_y` object. Each element corresponds to a crossfitting
+            fold and is the model instance that was fitted for that training fold.
+        """
         return [mdl._model for mdl in super().models_y]
 
     @property
     def models_t(self):
+        """
+        Get the fitted models for E[T | X, W].
+
+        Returns
+        -------
+        models_y: list of objects of type(`model_t`)
+            A list of instances of the `model_y` object. Each element corresponds to a crossfitting
+            fold and is the model instance that was fitted for that training fold.
+        """
         return [mdl._model for mdl in super().models_t]
 
+    def cate_feature_names(self, input_feature_names=None):
+        """
+        Get the output feature names.
+
+        Parameters
+        ----------
+        input_feature_names: list of strings of length X.shape[1] or None
+            The names of the input features
+
+        Returns
+        -------
+        out_feature_names: list of strings or None
+            The names of the output features :math:`\\phi(X)`, i.e. the features with respect to which the
+            final constant marginal CATE model is linear. It is the names of the features that are associated
+            with each entry of the :meth:`coef_` parameter. Available only when the featurizer is not None and has
+            a method: `get_feature_names(input_feature_names)`. Otherwise None is returned.
```

Comment:
This does not appear to correctly document the behavior - if the original featurizer is None we just return the input features unchanged (which is right, because those are the features we use in the final model)

Reply:
Done